front-end/src/components/Post/PostGovernanceInfo/PostReferendumInfo.tsx

```diff
@@ -39,12 +42,22 @@ const PostReferendumInfo = ({ onchainLink }: Props) => {
 				{(delay || delay === 0) &&
 					<Grid.Column mobile={16} tablet={8} computer={8}>
 						<h6>Delay</h6>
-						{delay}
+						<BlocksToTime blocks={delay} />
 					</Grid.Column>}
 				{end &&
 					<Grid.Column mobile={16} tablet={8} computer={8}>
-						<h6>End</h6>
-						{end}
+						{status === 'Started'
+							?
+							<>
+								<h6>Ends In</h6>
+								<BlockCountdown endBlock={end}/>
+							</>
+							:
+							<>
+								<h6>Started On</h6>
```

Comment:
I stupidly mistook `blockNumber.startDateTime` as the start of the referendum. It is actually the end date (compared it with the `end` value; it's the same). 

As every referendum other than "Started" is in the past, it only shows the timestamp on referenda in the past. It would be cleaner to also just use `end`,  but `end` is not of type `BlockNumber` so it doesn't have the `startDateTime` prop on it.

Reply:
I think what you did works well.